lib/signs/utilities/audio.ex

```diff
@@ -394,15 +394,29 @@ defmodule Signs.Utilities.Audio do
       sign,
       Enum.map(audios, &Content.Audio.to_params(&1)),
       Enum.map(audios, &Content.Audio.to_tts(&1)),
-      Enum.map(audios, fn audio ->
-        [
-          message_type: to_string(audio.__struct__) |> String.split(".") |> List.last(),
-          message_details: Map.from_struct(audio) |> inspect()
-        ]
-      end)
+      Enum.map(audios, &format_message/1)
     )
   end
 
+  # Format PA Messages separately to ensure backwards compatibility
+  defp format_message(%PaMessages.PaMessage{} = audio) do
+    message_details =
+      audio
+      |> Map.from_struct()
+      |> Map.drop([:sign_ids])
+      |> Enum.map(fn {k, v} -> {k, inspect(v)} end)
+
+    [message_type: to_string(audio.__struct__) |> String.split(".") |> List.last()] ++
+      message_details
+  end
+
+  defp format_message(audio) do
+    [
+      message_type: to_string(audio.__struct__) |> String.split(".") |> List.last(),
+      message_details: Map.from_struct(audio) |> inspect()
```

Comment:
**question:** did we want to expand the keys of this map and format the keys like we are for `%PaMessage.PaMessage{}` structs?

Reply:
I was reluctant to change these because I don't want to break any existing dashboards. This new function pattern is also the only one that produces logs we will use in our new dashboard.

Reply:
I think we set the expectation that those raw fields were for debugging purposes and might change, so dashboards shouldn't be built off them. Whether that's true is another matter, but Kevin would probably have an idea.

Reply:
I was wrong. This change doesn't have any effect on that dashboard. So we aren't using these logs yet and now is a good time to clean them up.